temoa/components/technology.py

```
@@ -431,3 +436,35 @@ def check_efficiency_variable(model: TemoaModel) -> None:
                 )
+
+
+def check_existing_capacity(model: TemoaModel) -> None:
+
+    for r, t, v in model.existing_capacity.sparse_iterkeys():
+
+        if t in model.tech_uncap:
+            msg = (
+                'Existing capacity entry for a tech without capacity (tech_unlim_cap set). '
+                'This is not allowed: ({}, {}, {})'
+            ).format(r, t, v)
```

Ruff (0.15.5)

[warning] Use f-string instead of `format` call

Convert to f-string

(UP032)